hudi-cli/src/main/java/org/apache/hudi/cli/commands/CommitsCommand.java

```diff
@@ -214,15 +224,18 @@ public String showArchivedCommits(
     }
   }
 
-  @CliCommand(value = "commits refresh", help = "Refresh the commits")
-  public String refreshCommits() throws IOException {
+  @CliCommand(value = {"commits refresh", "refresh"}, help = "Refresh the commits")
+  public String refreshCommits() {
     HoodieCLI.refreshTableMetadata();
     return "Metadata for table " + HoodieCLI.getTableMetaClient().getTableConfig().getTableName() + " refreshed.";
   }
 
   @CliCommand(value = "commit rollback", help = "Rollback a commit")
   public String rollbackCommit(@CliOption(key = {"commit"}, help = "Commit to rollback") final String instantTime,
-      @CliOption(key = {"sparkProperties"}, help = "Spark Properties File Path") final String sparkPropertiesPath)
+      @CliOption(key = {"sparkProperties"}, help = "Spark Properties File Path") final String sparkPropertiesPath,
+      @CliOption(key = "sparkMaster", unspecifiedDefaultValue = "", help = "Spark Master") String master,
+      @CliOption(key = "sparkMemory", unspecifiedDefaultValue = "4G",
```

Comment:
would the sparkMaster and sparkMemory put into sparkProperties? and if so, what's the priority?

Reply:
> would the sparkMaster and sparkMemory put into sparkProperties? and if so, what's the priority?

Usually put the parameters that are less changeable in `sparkProperties`, master and memory usually specified every time when run.